app/api/src/crud/crud_isochrone.py

```diff
@@ -441,36 +441,52 @@ async def calculate_pois_multi_isochrones(
         return isochrones_result
 
     async def export_isochrone(
-        self, db: AsyncSession, *, current_user, isochrone_calculation_id, return_type
+        self,
+        db: AsyncSession,
+        *,
+        current_user,
+        isochrone_calculation_id,
+        return_type,
+        geojson_dictionary: dict = None,
     ) -> Any:
+        if geojson_dictionary:
+            features = geojson_dictionary["features"]
+            # Remove the payload and set to true to use it later
+            geojson_dictionary = True
+            for _data in features:
+                # Shape the geometries
+                _data["geometry"] = shape(_data["geometry"])
+            gdf = GeoDataFrame(features).set_geometry("geometry")
+            gdf["minutes"] = gdf["properties"][0]["traveltime"]
+        else:
 
-        sql = text(
-            """
-            SELECT f.step AS seconds, f.id, c.modus, c.routing_profile, f.reached_opportunities, f.geom
-            FROM customer.isochrone_calculation c, customer.isochrone_feature f
-            WHERE c.id = f.isochrone_calculation_id
-            AND c.id = :isochrone_calculation_id
-            AND c.user_id = :user_id
-            """
-        )
+            sql = text(
```

Comment:
I think we can remove this part as we cannot read the information from the database as the data is not stored.